1147 - Operators:  use the "activate_bid" entrypoint (#1170)
casper-network/docs: source/docs/casper/operators/becoming-a-validator/bonding.md

## Method 2: Bonding with Compiled Wasm {#bonding-compiled-wasm}

> Another way to send a bonding transaction is to use the compiled `add_bid.wasm` and send it to the network via a deploy. For details, refer to [Building the Required Contracts](../setup/joining.md#step-3-build-contracts).

Reviewer: ```suggestion
Another way to send a bonding transaction to the network is via a deploy containing the compiled `add_bid.wasm`. For details, refer to [Building the Required Contracts](../setup/joining.md#step-3-build-contracts).
```

Something feels a little weird about this sentence structure, although I do think it's perfectly correct as is.

Author: I like the proposed suggestion better.
